.github/workflows/update-versions.yaml

```
@@ -11,6 +11,7 @@ on:
         type: string
 jobs:
   ocp-version:
+    if: github.repository == 'rh-ecosystem-edge/nvidia-ci' || ${{ vars.VERSION_UPDATE_FLOW_ENABLED }}
```

actionlint (1.7.4)

if: condition "github.repository == 'rh-ecosystem-edge/nvidia-ci' || ${{ vars.VERSION_UPDATE_FLOW_ENABLED }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)